### PR TITLE
Prep for Graphik

### PR DESCRIPTION
**Pull request by mperrotti**
Resolves issue #239 
Just need to include Graphik in the specified unicode ranges before we can finalize rebrand typography

### CODE REVIEW

File: sass/core/type/_headings.scss [hunk 1]
```diff
@@ -106,7 +94,53 @@ Headline text for when something between display and normal text is needed. Use
...
+/*doc
+---
+title: Header
+name: header
+parent: headings
+---
+Graphic text element used to label content. For example, used to display dates.
+
+```html_example
+<p class="text--headline">03</p>
```
Comment (akdetrick) [inline sass/core/type/_headings.scss]: looks like the doc example needs an update

File: sass/core/type/_headings.scss [hunk 2]
```diff
@@ -106,7 +94,53 @@ Headline text for when something between display and normal text is needed. Use
...
+<h2 class="header">Nov</h2>
+```
+*/
+%header,
```
Comment (akdetrick) [inline sass/core/type/_headings.scss]: we might want to go with the `text--[variant]` class style we've been using for text/heading classes

File: sass/core/type/_headings.scss [hunk 3]
```diff
@@ -106,7 +94,53 @@ Headline text for when something between display and normal text is needed. Use
...
+parent: headings
+---
+Lighter variant of `header`. For example, used in the redesign to label a MUP with a MUG name.
+
+```html_example
+<h2 class="header--secondary">Foreign Androids Feeling Frustrated</h2>
+<p class="text--bold">Building with Sassquatch2</p>
+```
+*/
+%header--secondary,
```
Comment (akdetrick) [inline sass/core/type/_headings.scss]: does `text--headerSecondary` work?

File: sass/core/type/_headings.scss (hunk 2 above)
Comment (mperrotti) [inline sass/core/type/_headings.scss]: We could, but I wanted `header` to be able to have more variants in the future. Right now we've just got `header--secondary` (which you propose to change to `headerSecondary` below), but I could see us having something like `header--accent`, etc.

I'm not sure how many variants we'll have, but we could just do `text--header--variant` or `text--headerVariant`

Comment (akdetrick) [inline sass/core/type/_headings.scss]: the consistency of all text classes starting with `text--` is nice - plus there's an implication of additive classes when you start with a single base class like `header`:

`class="button button--primary"`
`class="header header--variant"` - unless this is what we really want, i'd suggest...
`class="text--headerVariant"`

Comment (mperrotti) [inline sass/core/type/_headings.scss]: Ok, I'm on board with `text--headerVariant`
Making changes now!

**Pull request merged**